core/io/resource_importer.cpp

```diff
@@ -161,7 +161,7 @@ void ResourceFormatImporter::get_recognized_extensions(List<String> *p_extension
 void ResourceFormatImporter::get_recognized_extensions_for_type(const String &p_type, List<String> *p_extensions) const {
 
 	if (p_type == "") {
-		return get_recognized_extensions(p_extensions);
+		get_recognized_extensions(p_extensions);
```

Comment:
Doesn't this change the behavior by eliminating an early return?

Reply:
Yeah, that would change the logic of the program.
It is fixed now